RecoTracker/LST/test/LSTAlpakaTester.py

```diff
@@ -0,0 +1,134 @@
+
+## Timing
+#process.Timing = cms.Service("Timing")
+#process.options.wantSummary = cms.untracked.bool(True)
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1),
+    output = cms.optional.untracked.allowed(cms.int32,cms.PSet)
+)
+
+# Input source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/home/users/evourlio/TrackingNTupleProduction/CMSSW_12_6_0_pre2/src/step2_DIGI_L1TrackTrigger_L1_DIGI2RAW_HLT.root'),
+    #fileNames = cms.untracked.vstring('file:/home/users/phchang/work/lst/samples/CMSSW_12_5_0/src/step2_DIGI_L1TrackTrigger_L1_DIGI2RAW_HLT_PU.root'),
+    secondaryFileNames = cms.untracked.vstring()
+)
+
+process.options = cms.untracked.PSet(
+    FailPath = cms.untracked.vstring(),
+    IgnoreCompletely = cms.untracked.vstring(),
+    Rethrow = cms.untracked.vstring(),
+    SkipEvent = cms.untracked.vstring(),
+    accelerators = cms.untracked.vstring('*'),
+    allowUnscheduled = cms.obsolete.untracked.bool,
+    canDeleteEarly = cms.untracked.vstring(),
+    deleteNonConsumedUnscheduledModules = cms.untracked.bool(True),
+    dumpOptions = cms.untracked.bool(False),
+    emptyRunLumiMode = cms.obsolete.untracked.string,
+    eventSetup = cms.untracked.PSet(
+        forceNumberOfConcurrentIOVs = cms.untracked.PSet(
+            allowAnyLabel_=cms.required.untracked.uint32
+        ),
+        numberOfConcurrentIOVs = cms.untracked.uint32(0)
+    ),
+    fileMode = cms.untracked.string('FULLMERGE'),
+    forceEventSetupCacheClearOnNewRun = cms.untracked.bool(False),
+    makeTriggerResults = cms.obsolete.untracked.bool,
+    numberOfConcurrentLuminosityBlocks = cms.untracked.uint32(0),
+    numberOfConcurrentRuns = cms.untracked.uint32(1),
+    numberOfStreams = cms.untracked.uint32(0),
+    numberOfThreads = cms.untracked.uint32(1),
+    printDependencies = cms.untracked.bool(False),
+    sizeOfStackForThreadsInKB = cms.optional.untracked.uint32,
+    throwIfIllegalParameter = cms.untracked.bool(True),
+    wantSummary = cms.untracked.bool(False)
+)
+
+# Production Info
+process.configurationMetadata = cms.untracked.PSet(
+    annotation = cms.untracked.string('step3 nevts:10'),
+    name = cms.untracked.string('Applications'),
+    version = cms.untracked.string('$Revision: 1.19 $')
+)
+
+# Output definition
+
+process.output = cms.OutputModule("PoolOutputModule",
+    fileName = cms.untracked.string('LSTAlpakaTesterOutput.root'),
+    outputCommands = cms.untracked.vstring('keep *_lst*_*_*')
+)
+
+# Additional output definition
+
+# Other statements
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic_T21', '')
+
+# Tasks and Sequences for LST inputs
+process.load("RecoTracker.LST.lst_cff")
+process.load("Validation.RecoTrack.trackingNtuple_cff")
+lstInputTask = cms.Task(process.trackingNtupleSeedSelectors,process.siPhase2RecHits)
```

Comment:
@slava77
It is the `process.trackingNtupleSeedSelectors`, taken from the `trackingNtuple_cff`, that causes the HGCal to complain while `cmsRun`ing. Any idea why this is the case? I looked into it a bit, playing around with the configuration, but couldn't really tell without diving deep in all of the relevant configurations. And I am not sure if it's worth it...

Reply:
what were the warnings/errors from HGCAL? I forgot.

Reply:
It is a bunch of lines like these:
```
%MSG-w HGCalGeom:   HGCalGeometryESProducer:HGCalHESciGeometryESProducer@callESModule  31-Jan-2023 15:44:39 PST Run: 1 Event: 1
Check  HGCScintillatorDetId::EE:HE= 0:1 trigger= 0 type= 1 SiPM= 1 layer= 22 ring= 1:1 phi= 279:279 from  HGCScintillatorDetId::EE:HE= 0:1 trigger= 0 type= 1 SiPM= 1 layer= 22 ring= 1:1 phi= 279:279 ERROR ???
%MSG
%MSG-w HGCalGeom:   HGCalGeometryESProducer:HGCalHESciGeometryESProducer@callESModule  31-Jan-2023 15:44:39 PST Run: 1 Event: 1
Check  HGCScintillatorDetId::EE:HE= 0:1 trigger= 0 type= 1 SiPM= 1 layer= 22 ring= 1:1 phi= 280:280 from  HGCScintillatorDetId::EE:HE= 0:1 trigger= 0 type= 1 SiPM= 1 layer= 22 ring= 1:1 phi= 280:280 ERROR ???
%MSG
```

Reply:
It could be that something overrides something else when the trackingNtuple_cff is loaded here.
Normally in the ntuple tests this config is loaded at the end of the cfg file (via `--customise`), but I don't see anything obvious below that could depend on it and use HGCAL.

Another point, I don't quite understand why this is called at all.  
Can you add a tracer `process.Tracer = cms.Service("Tracer", printTimestamps = cms.untracked.bool(True))`?
Your setup is supposed to be trackingOnly, not running hgcal.

Reply:
> Another point, I don't quite understand why this is called at all.

Do you mean `process.trackingNtupleSeedSelectors` or the HGCal stuff?

I will add the Tracer. By the way, this seems to be called only at the beginning of processing the first event.

Reply:
I meant the HGCal stuff